Cargo.toml

```diff
@@ -129,4 +129,5 @@ crates_io_test_db = { path = "crates_io_test_db" }
 claims = "=0.7.1"
 googletest = "=0.10.0"
 insta = { version = "=1.34.0", features = ["json", "redactions"] }
+regex = "=1.10.2"
```

Comment:
I'm genuinely a little surprised this is the first time we've used `regex`.

(OK, a quick dig through Git suggests we had it until 2020.)